<ui version="4.0" >
 <class>cs8LoginDataDialogClass</class>
 <widget class="QDialog" name="cs8LoginDataDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>cs8 Login Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>IP Adress / Host Name of Controller</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="cbHostname" >
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="insertPolicy" >
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>User Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="leUserName" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="lePassword" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cs8LoginDataDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>277</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel" >
     <x>63</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cs8LoginDataDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>347</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>348</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
